server.js

```diff
@@ -598,8 +605,13 @@ proxy.on('proxyReq', function(proxyReq, req, res) {
 /**
  * Redirect to add the trailing slash without which mysterious bugs occur...
  */
-app.get(pathPrefix + '_design/' + db.settings.ddoc + '/_rewrite', function(req, res) {
-  res.redirect(appPrefix);
+[
+  appPrefix,
+  pathPrefix
+].forEach(function(url) {
+  app.get(url.substring(url.length - 1), function(req, res) {
```

Comment:
You could use charAt to check the last character is a slash? Also, using `slice` is a little prettier, eg: `url.slice(0, -1)`